README.md

```diff
@@ -5,6 +5,9 @@ https://github.com/FRINXio/FRINX-machine/releases
 
 > Master branch is used for development and not recommended for deployment
 
+</br>
+
+`For reusing .env file from Frinx-Machine 1.6 in Frinx-Machine 1.7 is need to update COMMON SETTINGS (krakend env)!`
```

Comment:
Is it detailed somewhere what to do in that case ? like a upgrade/migration section in Readme